source/SIL.AppBuilder.Portal.Frontend/src/translations/locales/en-us.json

```diff
@@ -73,7 +73,7 @@
     "emptyNotifications": "You have no notifications."
   },
   "sidebar": {
-    "myTasks": "My Tasks",
+    "myTasks": "My Tasks {count, plural, =0 {} other {({ count })}}",
```

Comment:
This looks bonkers but I assume its some magic with the translation library parsing to let you add parameters